common/src/main/scala/uk/ac/wellcome/platform/stacks/common/services/CatalogueService.scala

```diff
@@ -1,158 +1,137 @@
 package uk.ac.wellcome.platform.stacks.common.services
 
-import cats.instances.future._
-import cats.instances.list._
-import cats.syntax.traverse._
-import com.google.gson.internal.LinkedTreeMap
-import uk.ac.wellcome.platform.catalogue
-import uk.ac.wellcome.platform.catalogue.models.{ItemIdentifiers, ResultListItems}
-import uk.ac.wellcome.platform.stacks.common.models._
-
-import scala.collection.JavaConverters._
-import scala.concurrent.{ExecutionContext, Future}
-import scala.util.Try
-
-class CatalogueService(baseUrl: Option[String])(
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.{HttpRequest, Uri}
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import akka.stream.ActorMaterializer
+import uk.ac.wellcome.platform.stacks.common.models.{StacksItem, _}
+
+import scala.concurrent.{ExecutionContextExecutor, Future}
+
+object CatalogueService {
+
+  case class TypeStub(
+                       id: String,
+                       label: String
+                     )
+
+  case class LocationStub(
+                           locationType: TypeStub,
+                           label: Option[String],
+                           `type`: String
+                         )
+
+  case class IdentifiersStub(
+                              identifierType: TypeStub,
+                              value: String
+                            )
+
+  case class ItemStub(
+                       id: String,
+                       identifiers: List[IdentifiersStub],
+                       locations: List[LocationStub]
+                     )
+
+  case class WorkStub(
+                       id: String,
+                       items: List[ItemStub]
+                     )
+
+  case class WorkSearchResults(
+                                totalResults: Int,
+                                results: List[WorkStub]
+                              )
+
+}
+
+class CatalogueService(maybeBaseUri: Option[Uri])(
```

Comment:
The presence of the `Option` here feels a bit odd to me. Maybe set the default URI here? i.e.,

```scala
class CatalogueService(
  baseUri: Uri = Uri("https://api.wellcomecollection.org/catalogue/v2")
)
```

would simplify the calling code and anywhere that needs to use the base URI.